src/Makefile

```diff
@@ -564,7 +573,7 @@ help:
 	@echo "x86-64-sse3-popcnt      > x86 64-bit with sse3 and popcnt support"
 	@echo "x86-64-sse3             > x86 64-bit with sse3 support"
 	@echo "x86-64                  > x86 64-bit generic"
-	@echo "x86-32                  > x86 32-bit (also enables SSE)"
+	@echo "x86-32                  > x86 32-bit (also enables MMX and SSE)"
```

Comment:
Why not a x86-32-sse2 also?

Reply:
I did not want to modify the makefile too much, that's all.